portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +24,25 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
+      ArrayList<String> colleges = new ArrayList<String>() ;
+        
+      colleges.add("MIT");
+      colleges.add("Harvard");
+      colleges.add("NYU");
+             
+        
+     Gson gson = new Gson();
+     String json = gson.toJson(colleges);
+  
+
+
+    //response.setContentType("text/html;");
```

Comment:
Some old code here as well, might as well remove it since it's still store in git. If you want to see the history you can just check out the old version (which is magical).